CharacterGenerator/CharacterGenerator/COD4database.m

```diff
@@ -0,0 +1,138 @@
+//
+//  COD4database.m
+//  CharacterGenerator
+//
+//  Created by Z on 8/2/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "COD4database.h"
+
+@implementation COD4database
+
+-(void)setup{
+    [self setupPrimaryWeapons];
```

Comment:
I like that the setup is abstracted out into smaller helper instead of one very long method. That definitely makes this easier to read and understand 👍🏻